Motor/motors.go

```diff
@@ -52,7 +52,7 @@ func findFolder(port OutPort) string {
 		}
 	}
 
-	log.Fatal("No motor is connected to port %s", port)
+	log.Fatal("No motor is connected to port ", port )
```

Comment:
No. log.Fatal acts like Println not printf. 
It actually prints "No motor is connected to port %sA"
My correction prints "No motor is connected to port A"